search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -351,13 +351,33 @@
   [browse-image-related-url]
   (util/nil-if-value umm-spec-util/not-provided (:get-data-mime-type browse-image-related-url)))
 
+(defn- doi->distribution
+  "Returns a distribution open data field for the provided DOI.
+  See https://project-open-data.cio.gov/v1.1/schema/#dataset-distribution-fields."
+  [doi]
+  (when doi
+    {:accessURL (str "https://scholar.google.com/scholar?q=" (or (second
```

Comment:
We need to URL encode this URL. Maybe we should move the `ru/related-url->encoded-url` function somewhere common and reuse it.